remind.py

```diff
@@ -347,20 +347,28 @@ def _gen_msg(vevent, label):
         if hasattr(vevent, 'location') and vevent.location.value:
             msg.append('at %s' % vevent.location.value.strip())
 
-        if hasattr(vevent, 'description') and vevent.description.value:
+        has_desc = hasattr(vevent, 'description') and vevent.description.value
+
+        if tail or has_desc:
             rem.append('%%"%s%%"' % ' '.join(msg))
-            rem.append(vevent.description.value.strip())
         else:
             rem.append(' '.join(msg))
+        if tail:
+            rem.append(tail)
+
+        out = ' '.join(rem)
+        if has_desc:
+            out += sep + vevent.description.value.strip()
```

Comment:
I would propose to append the string to the list and move the join back to it's old place.